web-client/src/styles/custom.scss

```diff
@@ -2392,6 +2392,10 @@ button.change-scanner-button {
 	font-size: 32px;
 }
 
-.cities-notcalendared-header {
+.cities-not-calendared-header {
 	font-weight: 600;
+}
+
+.cities-not-calendated-yellow-background {
```

Comment:
```suggestion
.cities-not-calendared-yellow-background {
```

Reply:
Tiny tiny typo, not even worth fixing in this PR imo

Reply:
Will fix in main branch. Good catch and thank you.